samples/boards/nrf/system_off/src/main.c

```diff
@@ -4,29 +4,22 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include "retained.h"
```

Comment:
@nordicjm could that sample be moved to the retention API?

Reply:
It could be, and validity would be inbuilt, though would need overlays for each board